specs/altair/p2p-interface.md

```diff
@@ -94,6 +112,92 @@ Unaggregated signatures (along with metadata) are sent as `SyncCommitteeSignatur
 
 Aggregated sync committee signatures are packaged into (signed) `SyncCommitteeContribution` along with proofs and gossiped to the `sync_committee_contribution_and_proof` topic.
 
+### Transitioning the gossip
+
+With any fork, the fork version, and thus the `ForkDigestValue`, change.
+Message types are unique per topic, and so for a smooth transition a node will have to subscribe to both the old and new topics.
+
+All remaining topics are updated. In advance of the fork, a node SHOULD subscribe to the post-fork variants of the topics.
```

Comment:
```suggestion
All remaining topics are updated with the new `ForkDigestValue`. In advance of the fork, a node SHOULD subscribe to the post-fork variants of the topics.
```

this is a bit more clear imo

Reply:
I think the "remaining" part is confusing. It's more like "those topics names that stay in use after the fork update". I'll try to fix it